WrightTools/kit/_calculate.py

```diff
@@ -135,7 +138,7 @@ def nm_width(center, width, units="wn"):
     return red - blue
 
 
-def symmetric_sqrt(x, out=None):
+def symmetric_sqrt(x, out=None) -> np.ndarray:
```

Comment:
Union(np.ndarray, float, complex)

Either that or define a Generic x, and returns the type of x